common/src/main/java/io/netty/util/internal/logging/Log4J2LoggerFactory.java

```diff
@@ -19,6 +19,15 @@
 
 public final class Log4J2LoggerFactory extends InternalLoggerFactory {
 
+    public static final InternalLoggerFactory INSTANCE = new Log4J2LoggerFactory();
+
+    /**
+     * @deprecated Use {@link #INSTANCE} instead.
+     */
+    @Deprecated
+    public Log4J2LoggerFactory() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1186)